web.test.tests/web.test.tests/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace web.test.tests.Pages
+{
+    class SettingsPage
+    {
+        IWebDriver _driver;
+
+        public SettingsPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        const String XPathNumberFormatSelect = "//th[text()='Number format:']/following-sibling::td/select";
+        const String XPathCurrencySelect = "//th[text()='Default currency:']/following-sibling::td/select";
+        const String XPathDateFormatSelect = "//th[text()='Date format:']/following-sibling::td/select";
+
+        public IWebElement CancelBtn => _driver.FindElement(By.Id("cancel"));
+        public IWebElement Logout => _driver.FindElement(By.XPath("//div[text()='Logout']"));
+
+        public SelectElement SelectDateFormat => new SelectElement(_driver.FindElement(By.XPath(XPathDateFormatSelect)));
+        public SelectElement SelectNumberFormat => new SelectElement(_driver.FindElement(By.XPath(XPathNumberFormatSelect)));
+        public SelectElement SelectDefaultCurrency => new SelectElement(_driver.FindElement(By.XPath(XPathCurrencySelect)));
+
+        public void ClickSave()
+        {
+            _driver.FindElement(By.Id("save")).Click();
+            new WebDriverWait(_driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.AlertIsPresent());
+            IAlert alert = _driver.SwitchTo().Alert();
+            Assert.AreEqual("Changes are saved!", alert.Text);
+            alert.Accept();
```

Comment:
Don't do asserts in page object

Reply:
if (alert.Text == "Changes are saved!") alert.Accept();